docs/src/pages/components/breadcrumb.mdx

```diff
@@ -0,0 +1,63 @@
+---
+name: breadcrumb
+route: /components/breadcrumb
+---
+
+import Breadcrumb from '@pluralsight/ps-design-system-breadcrumb'
+
+import { Guideline, Types, Usage } from '../../components/component'
+
+# Breadcrumb
+
+<Intro>
+  The breadcrumb is to provide a consistently-placed and styled affordance to orient and move up a hierarchy.
+</Intro>
+
+## Examples
+
+The breadcrumb should be placed toward the top left of the page. Below any persistent heading or navigation elements.
+
+```typescript
+import Breadcrumb from '@pluralsight/ps-design-system-breadcrumb'
+import React from 'react'
+
+function Example() {
+  return (
+    <>
+      <Breadcrumb>All the things</Breadcrumb>
+    </>
+  )
+}
+
+export default Example
+```
+
+## Guidelines
+
+Reference the destination screen or page directly. Don't add text like "Back to..."
+
+<Guideline
+  do={<Breadcrumb>All things</Breadcrumb>}
+  dont={<Breadcrumb>Back to all things</Breadcrumb>}
+/>
+
+Breadcrumbs should be written in sentence case, except for proper nouns or product titles.
+
+<Guideline
+  do={<Breadcrumb>All the things</Breadcrumb>}
+  dont={<Breadcrumb>All The Things</Breadcrumb>}
+/>
+
+## Usage & Types
+
+<Usage
+  install="npm install @pluralsight/ps-design-system-breadcrumb"
+  import="import Breadcrumb from '@pluralsight/ps-design-system-breadcrumb'"
+  packageName="breadcrumb"
+/>
+
+<Types.Table>
+  <Types.Prop name="disabled" type="Boolean" desc="standard input disable flag" default="false" />
+  <Types.Prop name="innerRef" type="DOM element => ()" desc="react ref callback" />
```

Comment:
```suggestion
  <Types.Prop name="href" type="string" desc="target url; renders anchor" />
    <Types.Prop name="loading" type="boolean" desc="shows loading animation" />
    <Types.Prop name="onClick" type="Event => void" desc="click handler" />
```

You can remove this `innerRef` one.  It was removed in code and the docs are stale.

And I added a bunch in code that weren't in docs.